### ISSUE TITLE
Doesn't recognize downloaded dump file in file picker

### ISSUE THREAD

**Issue by NickSto** [gnosygnu/xowa#245]
Hi, I just installed XOWA on Ubuntu, and tried to import a dump file I'd already downloaded. But when I go to choose the file, it doesn't even appear in the file chooser.

However, I was able to paste the path into the text box, and it worked.

## Details

I went to `Dashboard/Import/Offline`, selected the correct wiki (Wikipedia/English), then in the "Where to get the dump" section, I chose "read from file", and clicked the little [...] button to choose the file. In the file chooser, I looked at the directories where I had several different database downloads:
- enwiki-20180820-pages-articles-multistream.xml.bz2
- enwiki-20160501-pages-articles.xml.bz2
- enwiki-20150112-pages-articles.xml.bz2

But none of them appeared.

Then, I pasted the complete path to the 2018 multistream file into the text box next to the [...] button, and hit "Import now", and it started importing.

## System
- Ubuntu 18.04.1 (64-bit)
  - running GNOME
- XOWA 4.5.21.1808 (64-bit)
- Oracle Java 1.8.0_181

**Comment by NickSto** [comment]
Hey, I'm back, just noting that this bug is still there in version 4.6.5.1911.
Still Ubuntu 18.04 (.5), and now OpenJDK 11.0.8.
I downloaded all 4 recommended files and put them in the same directory:
- enwiki-20200820-pages-articles.xml.bz2
- enwiki-20200820-categorylinks.sql.gz
- enwiki-20200820-page_props.sql.gz
- enwiki-20200820-pagelinks.sql.gz

None of them show up in the file picker.

**Comment by NijazM** [comment]
Yeah, I noticed this too on windows. Exploring files never worked in xowa. Only setting them in place via file manager in recommended location (xowa\wiki\en.wikipedia.org\ for english wikipedia) and using import starts import immediately. But never there was any choice or detection of those files as I thoght would be.
